packages/headless/src/features/facets/range-facets/date-facet-set/relative-date.ts

```diff
@@ -0,0 +1,94 @@
+import dayjs, {QUnitType} from 'dayjs';
+import {BooleanValue, NumberValue, Schema, StringValue} from '@coveo/bueno';
+import {formatDateForSearchApi} from './date-format';
+import utc from 'dayjs/plugin/utc';
+import quarterOfYear from 'dayjs/plugin/quarterOfYear';
+
+dayjs.extend(utc);
+dayjs.extend(quarterOfYear);
+
+const periodsObj = {past: 1, future: 1};
```

Comment:
Little trick to build the Type and the Array of constraints from the same source of truth